components/openweather_api/openweather_api.app.mjs

```diff
@@ -1,11 +1,121 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+
 export default {
   type: "app",
   app: "openweather_api",
-  propDefinitions: {},
+  propDefinitions: {
+    city: {
+      type: "string",
+      label: "City",
+      description: "City location. For example \"Houston\" in the United States.",
+    },
+    stateCode: {
+      type: "string",
+      label: "State code",
+      description: "State code. For example \"TX\" for Texas.",
+      optional: true,
+    },
```

Comment:
I think the `stateCode` should probably be required. I get an error if I test without entering a state.
```suggestion
    stateCode: {
      type: "string",
      label: "State code",
      description: "State code. For example \"TX\" for Texas.",
    },
```